### ISSUE TITLE
Cannot load AEGMM model from the example

### ISSUE THREAD

**Issue by mishinma** [SeldonIO/alibi-detect#80]
I have a problem running the "AEGMM and VAEGMM outlier detection on KDD Cup '99 dataset" notebook. For either model, after downloading its parameters from the cloud, loading fails with this error: 
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-9-7cf85ac01c21> in <module>
      1 filepath = './od_aegmm_kddcup/'  # change to directory where model is downloaded
      2 if load_outlier_detector:  # load pretrained outlier detector
----> 3     od = load_detector(filepath)
      4 else:  # define model, initialize, train and save outlier detector
      5     # the model defined here is similar to the one defined in the original paper

~/dev/alibi-detect/alibi_detect/utils/saving.py in load_detector(filepath)
    414 
    415     # load outlier detector specific parameters
--> 416     state_dict = pickle.load(open(os.path.join(filepath, detector_name + '.pickle'), 'rb'))
    417 
    418     # initialize outlier detector

ModuleNotFoundError: No module named 'odcd'
```

**Comment by arnaudvl** [comment]
The repo was originally called ```odcd```. I therefore suspect that the pretrained models were saved when the repo name hadn't changed yet. I will upload some newly trained models.

**Comment by mishinma** [comment]
Cool, thank you :)

**Comment by arnaudvl** [comment]
Issue addressed with #100
